AdhocTicket.java

```diff
@@ -0,0 +1,119 @@
+package bcccp.tickets.adhoc;
+
+import java.util.Date;
+
+public class AdhocTicket implements IAdhocTicket {
+	
+	public String carparkId;

```

Comment:
These variables need to remain private but variable name need to be changed

Reply:
Okay